en/docs/lib.md

```diff
@@ -1,11 +1,45 @@
 # Lib
 
-**Note:** This section is still a work in progress.
+Volt is a client and server framework. Gems exist for both client and server and many gems are not compatible with either client or server environment. This and other reasons (see below) mean that volt does not automatically require all gems at startup.
+
+### Gems for client and server
+
+Use Gemfile and bundler as always. But as mentioned above, you just have to remember to **require** them and eg. ```config/app.rb``` is a good place.
+
+### Client only gems
+
+A growing list of gems exist for use in the browser. Off course opal comes to mind, though that may still be run on any javascript vm.
+
+Opal-browser or opal-jquery are good examples, just like opal-pixi or opal-phantom, which all use the browser or dom api.
```

Comment:
Opal-browser ~~or~~ *and* opal-jquery...., ~~which all~~ *all of which* use... ...~~dom api~~ *DOM API*